chainer/optimizer.py

```diff
@@ -652,9 +655,58 @@ def serialize(self, serializer):
             if rule is not None:
                 rule.serialize(serializer[name])
 
+    def loss_scaling(self, interval=1000, scale=None):
+        """Sets loss scaling configurations.
+
+        Args:
+            interval (integer): Number of iterations until scaling factor gets
+                doubled. This is effective when "dynamic" loss scaling is used.
+            scale (float): Loss scaling factor. If None, "dynamic" loss scaling
+                is used, otherwise "static" loss scaling is used.
+        """
+        if scale is None:
+            self._loss_scaling_mode = 'dynamic'
+            if interval < 1:
+                raise ValueError("interval must be greater equal to 1")
```

Comment:
How about showing the actual value, e.g.
```suggestion
                raise ValueError("interval must be greater equal to 1. Actual: {}".format(interval))
```

Reply:
Thanks, that's better.

Reply:
Thanks!